src/app/store/telemetry-search.js

```diff
@@ -0,0 +1,44 @@
+import { readable } from 'svelte/store';
+
+import FlexSearch from 'flexsearch'
+const url = 'https://probeinfo.telemetry.mozilla.org/firefox/all/main/all_probes'
+
+const resultToProps = (info, id) => {
```

Comment:
@robhudson this function can probably be removed entirely since your ingestion job does this